tests/settings.py

```diff
@@ -173,3 +173,9 @@
 DEFAULT_HASHING_ALGORITHM = 'sha1'
 TEST_RUNNER = 'django.test.runner.DiscoverRunner'
 FIXTURE_DIRS = [location('unit/fixtures')]
+
+# Try and import local settings which can be used to override any of the above.
+try:
+    from tests.settings_local import *
```

Comment:
I think you  committed this by accident?

Reply:
I added this on purpose. How do you update the test database credentials?

Reply:
This is not something we would want in the test settings, when testing with different databases please add your own settings and don't push them to your branch.